### PR TITLE
Add troubleshooter for cricket

### PR DESCRIPTION
**Pull request by rich-nguyen**
Adds a page to admin for investigating issues with the PA cricket feed:

screen shot 2015-04-13 at 17 25 32

### CODE REVIEW

**Comment by gklopper** [comment]
:+1: - I will overlook the large scale indentation of the routes file...

File: admin/conf/routes [hunk 1]
```diff
@@ -3,128 +3,129 @@
 # ~~~~
```
Comment (robertberry) [inline admin/conf/routes]: Spacing gone crazeh

**Comment by robertberry** [comment]
:+1: otherwise

File: admin/conf/routes (hunk 1 above)
Comment (rich-nguyen) [inline admin/conf/routes]: Aah intellij! I must have pressed carriage-return on a routes file. fatal error.

**Pull request merged**